src/manual/endpoint-jms.adoc

```diff
@@ -600,6 +600,15 @@ In addition to that you need to add a client id on the connection factory so the
 subscription with the client address. Also we use the `SingleConnectionFactory` implementation of Spring as a connection factory
 wrapper so we do not fail because of multiple connections with the same durable subscriber id.
 
+[[jms-topic-purging]]
+== JMS topic purging
+As mentioned earlier, topic subscribers with `autoStart=true` consume all messages sent to the topic during the test run.
```

Comment:
just a nitpick: please insert a blank line after the heading, because some Asciidoc parsers for website generation do not like that and may not recognize the section heading properly. Many thanks

Reply:
```suggestion

As mentioned earlier, topic subscribers with `autoStart=true` consume all messages sent to the topic during the test run.
```